README.rst

```diff
@@ -1422,6 +1463,74 @@ Example usage:
         '*png',
     ]
 
+``OPENWISP_HTML_EMAIL``
+^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+----------+
+| type    | ``bool`` |
++---------+----------+
+| default | ``True`` |
++---------+----------+
+
+If ``True``, attaches markdown rendered HTML of message in email.
+If ``False``, HTML rendering of message will be disabled and a plain
+text email is sent.
+
+``OPENWISP_EMAIL_TEMPLATE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+----------------------------------------+
+| type    | ``str``                                |
++---------+----------------------------------------+
+| default | ``openwisp_utils/email_template.html`` |
++---------+----------------------------------------+
+
+This setting takes the path to the template for email. Thus, making it possible to
+customize email. You can either extend the default email template or write your own
+email template from scratch. An example of extending default email template to customize styling is
+shown below.
```

Comment:
I'd simplify this to something like:

This setting allows to change the django template used for sending emails with the send_email function (add a link to the send email function here). It is recommended to extend the default email template as in the example below.